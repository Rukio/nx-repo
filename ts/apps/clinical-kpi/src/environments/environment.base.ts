export type ClinicalKPIEnvironment = {
  production: boolean;
  stationURL: string;
  auth0Domain: string;
  auth0Audience: string;
  auth0ClientId: string;
  statsigClientKey: string;
  statsigTier: string;
  datadogApplicationId: string;
  datadogClientToken: string;
  datadogEnvironment: 'development' | 'qa' | 'uat' | 'production';
};
