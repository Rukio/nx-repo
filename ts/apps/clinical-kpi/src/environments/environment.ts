import { ClinicalKPIEnvironment } from './environment.base';

export const environment: ClinicalKPIEnvironment = {
  production: false,
  stationURL: 'https://qa.*company-data-covered*.com/',
  auth0Domain: 'staging-auth.*company-data-covered*.com',
  auth0Audience: 'internal.*company-data-covered*.com',
  auth0ClientId: 'jwdBIvE9iZj9ODuSBd8kbqjcXb9S1gQ4',
  statsigClientKey: 'client-wQHgyJwKCkbJVxVItgwrQGfaxk7hglbFBSB7YuhZl5z',
  statsigTier: 'development',
  datadogApplicationId: 'dacc30dd-49bf-441a-b633-2771f801d618',
  datadogClientToken: 'pub1f5e0ea7cae0554165ac74a3ce93b1d2',
  datadogEnvironment: 'development',
};
