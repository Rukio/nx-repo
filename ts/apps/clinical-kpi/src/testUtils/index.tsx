import { ReactElement } from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render, RenderOptions } from '@testing-library/react';
import { StoreProvider } from '@*company-data-covered*/clinical-kpi/feature';

type CustomRenderOptions = {
  options?: Omit<RenderOptions, 'queries'>;
  withRouter?: boolean;
};

const customRender = (
  ui: ReactElement,
  { options, withRouter = false }: CustomRenderOptions = {}
) => {
  const component = withRouter ? <BrowserRouter>{ui}</BrowserRouter> : ui;

  return render(<StoreProvider>{component}</StoreProvider>, options);
};

export * from '@testing-library/react';
export { customRender as render };
