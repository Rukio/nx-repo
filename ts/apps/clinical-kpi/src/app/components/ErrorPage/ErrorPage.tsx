import { AlertButton } from '@*company-data-covered*/clinical-kpi/ui';
import { AppBar } from '@*company-data-covered*/clinical-kpi/feature';

import { environment } from '../../../environments/environment';

const ErrorPage = () => {
  return (
    <>
      <AppBar stationURL={environment.stationURL} />
      <AlertButton
        text="Looks like something went wrong. Please check back later."
        buttonLink={environment.stationURL}
        buttonText="Continue to Dashboard"
      />
    </>
  );
};

export default ErrorPage;
