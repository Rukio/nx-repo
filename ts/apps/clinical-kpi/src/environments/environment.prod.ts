import { ClinicalKPIEnvironment } from './environment.base';

export const environment: ClinicalKPIEnvironment = {
  production: true,
  stationURL: 'https://admin.*company-data-covered*.com/',
  auth0Domain: 'auth.*company-data-covered*.com',
  auth0Audience: 'internal.*company-data-covered*.com',
  auth0ClientId: 'onOW70ihQJ3JOKPKx5MSY7aSTznithTg',
  statsigClientKey: 'client-heX0sQN8B3HKxmUUkInqqdtYEy8cENb38GlCFVjF6Cn',
  statsigTier: 'production',
  datadogApplicationId: 'dacc30dd-49bf-441a-b633-2771f801d618',
  datadogClientToken: 'pub1f5e0ea7cae0554165ac74a3ce93b1d2',
  datadogEnvironment: 'production',
};
