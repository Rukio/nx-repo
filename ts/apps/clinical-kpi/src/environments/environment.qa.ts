import { ClinicalKPIEnvironment } from './environment.base';

export const environment: ClinicalKPIEnvironment = {
  production: true,
  stationURL: 'https://qa.*company-data-covered*.com/',
  auth0Domain: 'staging-auth.*company-data-covered*.com',
  auth0Audience: 'internal.*company-data-covered*.com',
  auth0ClientId: 'jwdBIvE9iZj9ODuSBd8kbqjcXb9S1gQ4',
  statsigClientKey: 'client-cY8wRPcwjrSnuHxGSdCU9ODaFpn01OMrInfxxlC08Pl',
  statsigTier: 'qa',
  datadogApplicationId: 'dacc30dd-49bf-441a-b633-2771f801d618',
  datadogClientToken: 'pub1f5e0ea7cae0554165ac74a3ce93b1d2',
  datadogEnvironment: 'qa',
};
