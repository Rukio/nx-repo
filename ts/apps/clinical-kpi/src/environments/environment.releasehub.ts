import { ClinicalKPIEnvironment } from './environment.base';

export const environment: ClinicalKPIEnvironment = {
  production: false,
  stationURL: process.env['NX_CLINICALKPI_DASHBOARD_STATION_URL'] ?? '',
  auth0Domain:
    process.env['NX_CLINICALKPI_DASHBOARD_AUTH0_DOMAIN'] ??
    'staging-auth.*company-data-covered*.com',
  auth0Audience:
    process.env['NX_CLINICALKPI_DASHBOARD_AUTH0_AUDIENCE'] ??
    'internal.*company-data-covered*.com',
  auth0ClientId:
    process.env['NX_CLINICALKPI_DASHBOARD_AUTH0_CLIENT_ID'] ??
    'jwdBIvE9iZj9ODuSBd8kbqjcXb9S1gQ4',
  statsigClientKey:
    process.env['NX_CLINICALKPI_DASHBOARD_STATSIG_CLIENT_KEY'] ??
    'client-wQHgyJwKCkbJVxVItgwrQGfaxk7hglbFBSB7YuhZl5z',
  statsigTier:
    process.env['NX_CLINICALKPI_DASHBOARD_AUTH0_STATSIG_TIER'] ?? 'development',
  datadogApplicationId: 'dacc30dd-49bf-441a-b633-2771f801d618',
  datadogClientToken: 'pub1f5e0ea7cae0554165ac74a3ce93b1d2',
  datadogEnvironment: 'development',
};
