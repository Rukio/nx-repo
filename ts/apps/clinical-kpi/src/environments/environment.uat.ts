import { ClinicalKPIEnvironment } from './environment.base';

export const environment: ClinicalKPIEnvironment = {
  production: true,
  stationURL: 'https://uat.*company-data-covered*.com/',
  auth0Domain: 'dispatch-uat.us.auth0.com',
  auth0Audience: 'internal.*company-data-covered*.com',
  auth0ClientId: 'TcD4VBbip7zUtZrBeCgKnFjQjkZZ2mbW',
  statsigClientKey: 'client-FWJJtvtaJ5Pkki0v9x4U5gsURDiTDodYUEAuXhLEKu2',
  statsigTier: 'uat',
  datadogApplicationId: 'dacc30dd-49bf-441a-b633-2771f801d618',
  datadogClientToken: 'pub1f5e0ea7cae0554165ac74a3ce93b1d2',
  datadogEnvironment: 'uat',
};
