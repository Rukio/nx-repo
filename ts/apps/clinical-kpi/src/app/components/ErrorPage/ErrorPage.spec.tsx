import { render, screen } from '../../../testUtils';
import ErrorPage from './ErrorPage';
import { TEST_IDS } from '@*company-data-covered*/clinical-kpi/ui';
import { environment } from '../../../environments/environment';

jest.mock('statsig-js', () => ({
  checkGate: jest.fn(),
}));

describe('ErrorPage', () => {
  it('should render error page correctly', () => {
    const { asFragment } = render(<ErrorPage />, { withRouter: true });
    expect(asFragment()).toMatchSnapshot();
    expect(
      screen.getByTestId(TEST_IDS.ALERT_BUTTON.TEXT).textContent
    ).toContain('Looks like something went wrong. Please check back later.');
    expect(
      screen.getByTestId(TEST_IDS.ALERT_BUTTON.BUTTON).textContent
    ).toContain('Continue to Dashboard');
    expect(
      screen.getByTestId(TEST_IDS.ALERT_BUTTON.BUTTON).getAttribute('href')
    ).toContain(environment.stationURL);
  });
});
