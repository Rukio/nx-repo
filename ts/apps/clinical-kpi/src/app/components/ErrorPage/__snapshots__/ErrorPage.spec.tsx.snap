// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage should render error page correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  width: 100%;
  margin-left: auto;
  box-sizing: border-box;
  margin-right: auto;
  display: block;
  padding-left: 16px;
  padding-right: 16px;
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  padding-top: 32px;
}

@media (min-width:600px) {
  .emotion-0 {
    padding-left: 24px;
    padding-right: 24px;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    max-width: 1200px;
  }
}

.emotion-1 {
  height: 1rem;
  width: 100%;
  line-height: 1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
}

@media (max-width:599.95px) {
  .emotion-1 {
    -webkit-box-pack: justify;
    -webkit-justify-content: space-between;
    justify-content: space-between;
  }
}

.emotion-2 {
  margin: 0;
  color: #1976d2;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  text-decoration-color: rgba(25, 118, 210, 0.4);
}

.emotion-2:hover {
  text-decoration-color: inherit;
}

.emotion-3 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.8125rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 4px 5px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  color: #1976d2;
  margin-left: 40px;
  text-transform: none;
  color: #1976d2;
}

.emotion-3::-moz-focus-inner {
  border-style: none;
}

.emotion-3.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-3 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-3:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: rgba(25, 118, 210, 0.04);
}

@media (hover: none) {
  .emotion-3:hover {
    background-color: transparent;
  }
}

.emotion-3.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
}

.emotion-4 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.5;
  letter-spacing: 0.00938em;
}

@media (max-width:599.95px) {
  .emotion-4 {
    display: none;
  }
}

.emotion-5 {
  display: inherit;
  margin-right: -2px;
  margin-left: 8px;
}

.emotion-5>*:nth-of-type(1) {
  font-size: 18px;
}

.emotion-6 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
}

.emotion-7 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

.emotion-8 {
  position: absolute;
  right: 0;
  top: 0;
}

@media (max-width:599.95px) {
  .emotion-8 {
    display: none;
  }
}

<div
    class="MuiContainer-root MuiContainer-maxWidthLg emotion-0"
  >
    <nav
      class="MuiBox-root emotion-1"
    >
      <a
        class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways emotion-2"
        data-testid="appbar-dispatch-health-logo-link"
        href="https://qa.*company-data-covered*.com/"
      >
        <svg>
          *company-data-covered*-logo.svg
        </svg>
      </a>
      <a
        class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeSmall MuiButton-textSizeSmall emotion-3"
        data-testid="appbar-station-dashboard-link"
        href="https://qa.*company-data-covered*.com/"
        tabindex="0"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 emotion-4"
          data-testid="appbar-station-dashboard-link-label-prefix"
        >
          Continue toÂ  Dashboard
        </p>
        <span
          class="MuiButton-endIcon MuiButton-iconSizeSmall emotion-5"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium emotion-6"
            data-testid="ArrowForwardIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="m12 4-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
            />
          </svg>
        </span>
        <span
          class="MuiTouchRipple-root emotion-7"
        />
      </a>
      <div
        class="MuiBox-root emotion-8"
      >
        <svg
          data-testid="appbar-kpi-dashboard-graphic"
        >
          kpidashboard-graphic.svg
        </svg>
      </div>
    </nav>
  </div>
  .emotion-0 {
  width: 100%;
  margin-left: auto;
  box-sizing: border-box;
  margin-right: auto;
  display: block;
  padding-left: 16px;
  padding-right: 16px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding-top: 5%;
  background-color: inherit;
}

@media (min-width:600px) {
  .emotion-0 {
    padding-left: 24px;
    padding-right: 24px;
  }
}

@media (min-width:1200px) {
  .emotion-0 {
    max-width: 1200px;
  }
}

.emotion-1 {
  background-color: #fff;
  color: rgba(0, 0, 0, 0.87);
  -webkit-transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: box-shadow 300ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  border-radius: 4px;
  box-shadow: none;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 0.875rem;
  line-height: 1.43;
  letter-spacing: 0.01071em;
  background-color: transparent;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  padding: 6px 16px;
  color: rgb(30, 70, 32);
  border: 1px solid #4caf50;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  text-align: center;
  width: 300px;
  background-color: inherit;
}

.emotion-1 .MuiAlert-icon {
  color: #4caf50;
}

.emotion-2 {
  margin-right: 12px;
  padding: 7px 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 22px;
  opacity: 0.9;
}

.emotion-3 {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  width: 1em;
  height: 1em;
  display: inline-block;
  fill: currentColor;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  -webkit-transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: fill 200ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  font-size: 1.5rem;
  color: rgba(0, 0, 0, 0.54);
  scale: 1.5;
}

.emotion-4 {
  padding: 8px 0;
  min-width: 0;
  overflow: auto;
}

.emotion-5 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.5;
  letter-spacing: 0.00938em;
  color: rgba(0, 0, 0, 0.6);
}

.emotion-6 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 500;
  font-size: 0.9375rem;
  line-height: 1.75;
  letter-spacing: 0.02857em;
  text-transform: uppercase;
  min-width: 64px;
  padding: 8px 22px;
  border-radius: 4px;
  -webkit-transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  color: #fff;
  background-color: #1976d2;
  box-shadow: 0px 3px 1px -2px rgba(0,0,0,0.2),0px 2px 2px 0px rgba(0,0,0,0.14),0px 1px 5px 0px rgba(0,0,0,0.12);
}

.emotion-6::-moz-focus-inner {
  border-style: none;
}

.emotion-6.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-6 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-6:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: #1565c0;
  box-shadow: 0px 2px 4px -1px rgba(0,0,0,0.2),0px 4px 5px 0px rgba(0,0,0,0.14),0px 1px 10px 0px rgba(0,0,0,0.12);
}

@media (hover: none) {
  .emotion-6:hover {
    background-color: #1976d2;
  }
}

.emotion-6:active {
  box-shadow: 0px 5px 5px -3px rgba(0,0,0,0.2),0px 8px 10px 1px rgba(0,0,0,0.14),0px 3px 14px 2px rgba(0,0,0,0.12);
}

.emotion-6.Mui-focusVisible {
  box-shadow: 0px 3px 5px -1px rgba(0,0,0,0.2),0px 6px 10px 0px rgba(0,0,0,0.14),0px 1px 18px 0px rgba(0,0,0,0.12);
}

.emotion-6.Mui-disabled {
  color: rgba(0, 0, 0, 0.26);
  box-shadow: none;
  background-color: rgba(0, 0, 0, 0.12);
}

.emotion-7 {
  overflow: hidden;
  pointer-events: none;
  position: absolute;
  z-index: 0;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  border-radius: inherit;
}

<div
    class="MuiContainer-root MuiContainer-maxWidthLg emotion-0"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 MuiAlert-root MuiAlert-outlinedSuccess MuiAlert-outlined emotion-1"
      role="alert"
    >
      <div
        class="MuiAlert-icon emotion-2"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-colorAction MuiSvgIcon-fontSizeMedium emotion-3"
          data-testid="ErrorOutlineIcon"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
          />
        </svg>
      </div>
      <div
        class="MuiAlert-message emotion-4"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 emotion-5"
          data-testid="alert-button-text"
        >
          Looks like something went wrong. Please check back later.
        </p>
      </div>
    </div>
    <a
      class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeLarge MuiButton-containedSizeLarge emotion-6"
      data-testid="alert-button-button"
      href="https://qa.*company-data-covered*.com/"
      tabindex="0"
    >
      Continue to Dashboard
      <span
        class="MuiTouchRipple-root emotion-7"
      />
    </a>
  </div>
</DocumentFragment>
`;
